conmon/conmon.c

```diff
@@ -93,25 +93,95 @@ static GOptionEntry entries[] =
   { NULL }
 };
 
-int set_k8s_timestamp(char *buf, ssize_t buflen, const char *stream_type)
+/* strlen("1997-03-25T13:20:42+01:00") + 1 */
```

Comment:
@mikebrow Yeah, we can do that for sure. Just want to get this correctness patch in first :)